pkgs/by-name/za/zapret/package.nix

```
@@ -15,38 +15,44 @@
 
 stdenv.mkDerivation {
   pname = "zapret";
-  version = "0-unstable-2024-08-01";
+  version = "65.1";
 
   src = fetchFromGitHub {
     owner = "bol-van";
     repo = "zapret";
-    rev = "9cf72b7c68f6a7c80dfddc6c1cf6d6db32718376";
-    hash = "sha256-8cqKCNYLLkZXlwrybKUPG6fLd7gmf8zV9tjWoTxAwIY=";
+    rev = "refs/tags/v65.1";
+    hash = "sha256-Wd9OqbhQwD7Lbi4zBcfgFSPvi8/K4G+xPso3VXtXdAI=";
   };
 
   buildInputs = [ libcap zlib libnetfilter_queue libnfnetlink ];
   nativeBuildInputs = [ iptables nftables gawk ];
 
   buildPhase = ''
+    runHook preBuild
+
     mkdir -p $out/bin
 
     make TGT=$out/bin
```

```suggestion
  makeFlags = [ "TGT=${placeholder "out"}" ];
```

And remove `buildPhase`. But it would be better to set TGT to some temporary directory instead, or `.`, so we aren't mixing up phases.